src/server.h

```diff
@@ -112,6 +113,16 @@ class CHighPrecisionTimer : public QThread
 };
 #endif
 
+// Minimal int16_t wrapper for queuing
```

Comment:
Can you explain in short words why you need an additional class here?

Reply:
Because `int16_t` is a `#define` not a type.  The queuing declarations required real types, as far as I could tell.  I _think_ I could have used `short` - but that would then have been inconsistent with the datatype for the value in question.